vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -331,7 +333,8 @@ class MessageActionsViewModel @AssistedInject constructor(
                 add(EventSharedAction.Reply(eventId))
             }
 
-            if (canReplyInThread(timelineEvent, messageContent, actionPermissions)) {
+            if (canReplyInThread(timelineEvent, messageContent, actionPermissions) &&
```

Comment:
I would prefer to return false in canReplyInThread instead of injecting the boolean (there is already a line to return a boolean according to the threads availability which has been commented)

Reply:
I'm not agree with that, canReplyInThread is come from element with their specifications.
We already did like this below l350.
I prefer to have the same tchap modification everywhere.

Reply:
I understand your point of view.
But the threads feature is still activable in debug from the labs settings, so I'd like to have the quick action available in the case that the feature has been activated.